api/types/plugin.go

```diff
@@ -25,10 +25,6 @@ type Plugin struct {
 	// settings
 	// Required: true
 	Settings PluginSettings `json:"Settings"`
-
-	// tag
-	// Required: true
-	Tag string `json:"Tag"`
```

Comment:
I see that daemon/cluster/executor/container/executor.go has been updated in this PR. SwarmKit will need a similar change. Making a note here so we don't forget.